docker-compose.yml

```diff
@@ -28,12 +28,26 @@ services:
       timeout: 5s
       retries: 3
 
+  migrations:
+    image: mtsrus/syncmaster-backend:${TAG:-develop}
+    restart: none
+    build:
+      dockerfile: docker/Dockerfile.backend
```

Comment:
is this intentionally, choosing a backend image while `docker-compose.test.yml` uses a more lightweight version with a worker for migrations?

Reply:
> docker-compose.test.yml uses a more lightweight version with a worker for migrations

Worker image is not very lightweight as it includes Java and Spark. I'm using worker image in CI to run migrations, just to avoid building backend image (this increases CI time). 

We'll probably update worker to call API instead of connecting to database directly, so migrations definitely shouldn't be included here.

Reply:
Added comment to `docker-compose.test.yml`